lib/pages/share/share.dart

```diff
@@ -42,6 +48,8 @@ class ShareController extends State<Share> with SendFilesMixin {
     isShowRecentlyChatsNotifier.value = !isShowRecentlyChatsNotifier.value;
   }
 
+  void toggleSearchMode() {}
```

Comment:
empty ?

Reply:
yeah, current not support search, but i reuse the old component, so the method need to have to be able to reused